local_settings_example.py

```diff
@@ -469,6 +471,25 @@
     SAML_DJANGO_USER_MAIN_ATTRIBUTE = 'username'
     SAML_DJANGO_USER_MAIN_ATTRIBUTE_LOOKUP = '__iexact'
 
+    # The list of IdPs we talk to are defined
```

Comment:
```suggestion
```

Reply:
Ooops. Good catch. I moved the IDP definition outside since I couldn't get flake8 to stop complaining about it.